mex/confluence_vvt/transform.py

```diff
@@ -7,13 +7,13 @@
 
 
 @watch
-def transform_confluence_vvt_sources_to_mex_sources(
+def transform_confluence_vvt_sources_to_mex_activities(
```

Comment:
```suggestion
def transform_confluence_vvt_sources_to_extracted_activities(
```

maybe even this, while we're at it

Reply:
done in 01db3ac38a89886407e5d6c1f5228dae6d3b8377